flutter_local_notifications/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Sat Jul 04 21:26:33 IDT 2020
+distributionBase=GRADLE_USER_HOME
```

Comment:
Can you revert adding this in? My understanding is that this would start locking the plugin with specific Gradle distribution versions. A number of the first-party plugins leave this out too

Reply:
100% right